features/src/autogluon/features/generators/text_ngram.py

```diff
@@ -60,9 +61,19 @@ def __init__(self, vectorizer=None, vectorizer_strategy='combined', max_memory_r
             raise ValueError(f"vectorizer_strategy must be one of {['combined', 'separate', 'both']}, but value is: {vectorizer_strategy}")
         self.vectorizer_strategy = vectorizer_strategy
         self.vectorizer_features = None
+        self.prefilter_tokens = prefilter_tokens
+        self.prefilter_token_count = prefilter_token_count
+        self.token_mask = None
 
-    def _fit_transform(self, X: DataFrame, **kwargs) -> (DataFrame, dict):
+    def _fit_transform(self, X: DataFrame, y: Series, **kwargs) -> (DataFrame, dict):
```

Comment:
Please add a unit test showcasing these changes that also tests edge cases.

Reply:
need some help with this. what exactly should we test for?

Reply:
Tests should be added to this file: https://github.com/awslabs/autogluon/blob/master/tabular/tests/unittests/features/generators/test_text_ngram.py

Essentially, we want to test that using the functionality works in the way we expect it to. Edge cases include if y contains NaN values in rows (A valid situation if we passed concatenated train and unlabeled data to increase number of ngrams we generate) and k > ngram count.